app/api/server/v1/invites.js

```diff
@@ -0,0 +1,57 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../api';
+import { findOrCreateInvite } from '../../../invites/server/functions/findOrCreateInvite';
+import { listInvites } from '../../../invites/server/functions/listInvites';
+import { useInviteToken } from '../../../invites/server/functions/useInviteToken';
+import { validateInviteToken } from '../../../invites/server/functions/validateInviteToken';
+
+API.v1.addRoute('listInvites', { authRequired: true }, {
+	get() {
+		let result;
+		Meteor.runAsUser(this.userId, () => {
+			result = listInvites(this.userId);
+		});
```

Comment:
```javascript
		const result = Meteor.runAsUser(this.userId, () => listInvites(this.userId));
```

Reply:
I thinks there is no need to use `Meteor.runAsUser` at all..

Reply:
That's true since it's not calling any Meteor method